packages/faceted-search/src/components/Badges/BadgeOverlay/BadgeOverlay.component.js

```diff
@@ -68,8 +74,13 @@ const BadgeOverlay = ({
 			type="button"
 			ref={target => setButtonRef(target)}
 			onClick={changeOpened}
+			className={theme({ 'tc-badge-link-plus': hasAddButton })}
 		>
-			{iconName ? <Icon name={`talend-${iconName}`} key="icon" /> : getLabel(label)}
+			{iconName && (
```

Comment:
If another use case pop in the future, this part may need a refacto to avoid too much condition in the render.
Maybe use directly a props that handle the button content or specialize version of badge overlay (like a AddBadge component).